src/Cake.Common.Tests/Unit/Tools/NuGet/Pack/NuGetPackerTests.cs

```diff
@@ -593,7 +822,7 @@ public void Should_Replace_Template_Tokens_In_Nuspec_With_Files_And_DependencyTa
 
                     // Then
                     Assert.Equal(
-                        Resources.Nuspec_Metadata_WithTragetFramworkDependencies.NormalizeLineEndings(),
+                        Resources.Nuspec_Metadata_WithTragetFrameworkDependencies.NormalizeLineEndings(),
```

Comment:
Can you fix the other spelling error here while you're at it? ('Target')

Reply:
Done.